tests/end-to-end/tests/logo-cloud.cy.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2022 Dynamic Solutions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { selectors, testIds } from '../support/consts';
+
+const paths = {
+  logoCloud:
+    'ComponentOverlay_rootcontainer/maincontainer/pagesection/logoscloud',
+  logoCloudImage:
+    'ComponentOverlay_rootcontainer/maincontainer/pagesection/logoscloud/image1'
+};
+
+describe('Logo cloud component', function () {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('renders correctly in preview mode', function () {
+    cy.visit('/content/howlite-test/pages/Logo-cloud.html');
+    cy.percySnapshotWithAuth('Logo cloud preview');
+  });
+
+  it('renders correctly in edit mode', function () {
+    cy.visit(
+      '/apps/websight/index.html/content/howlite-test/pages/Logo-cloud::editor'
+    );
+
+    cy.percySnapshotWithAuth('Logo cloud editor');
+
+    cy.intercept(
+      'POST',
+      '**/pagesection/logoscloud/image1.websight-dialogs-service.save-properties.action'
+    ).as('saveProperties');
+
+    cy.getByTestId(paths.logoCloud)
+      .click({force: true})
+      .find(selectors.overlayName)
+      .should('have.text', 'Logo cloud');
+
+    cy.getByTestId(paths.logoCloudImage)
+      .click()
+      .find(selectors.overlayName)
+      .should('have.text', 'Image');
+
+    cy.getByTestId(testIds.editIcon).click();
+
+    cy.percySnapshotWithAuth('Logo cloud image dialog');
```

Comment:
we don't need to test image component (it should be tested in image.cy instead). I think all we could do here is:
- open logo cloud dialog
- take s percysnapshot
- change grid values to for example: 10 cols width, 1 col offset (let's wait for grid PR to be merged)
- save
- check json file

Reply:
Alright, I'm waiting for this fix until your grid PR will be merged.

Reply:
@marcell-pal-ds I've just merged new grid so we can use offset properties now (smOffset, mdOffset and lgOffset)